IPython/terminal/ptutils.py

```diff
@@ -12,8 +12,12 @@
 
 class IPythonPTCompleter(Completer):
     """Adaptor to provide IPython completions to prompt_toolkit"""
-    def __init__(self, ipy_completer):
-        self.ipy_completer = ipy_completer
+    def __init__(self, shell):
```

Comment:
**gruml gluml** change of API.. **gruml, gruml** but I see the reason. 
Can we check the type of shell and print a deprecation warning if it's a Completer  ?